### PR TITLE
Ardupilot Gazebo Basic Support Update

### PR DESCRIPTION
**Pull request by SwiftGust**
Added gazebo worlds, demo worlds from unmerged bitbucket PRs.
Now it is possible to configure gazebo with ardupilot by simply cloning this repository.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -39,7 +39,9 @@ link_directories(
 set (plugins_single_header
         ArduPilotPlugin
         ArduCopterIRLockPlugin
+	ArduCopterPlugin
         GimbalSmall2dPlugin
+	LiftDragPlugin
```
Comment (khancyr) [inline CMakeLists.txt]: liftdrag is already in gazebo starting from 7.0

File: CMakeLists.txt [hunk 2]
```diff
@@ -39,7 +39,9 @@ link_directories(
 set (plugins_single_header
         ArduPilotPlugin
         ArduCopterIRLockPlugin
+	ArduCopterPlugin
```
Comment (khancyr) [inline CMakeLists.txt]: I won't merge this as ardupilot one is better

**Comment by khancyr** [comment]
Well there are plenty of thing here ! I will start by merging the readme changes as they are the most usefull . For the model, I will make a clean up ! Having all model isn't necessary and is very heavy .

File: CMakeLists.txt (hunk 2 above)
Comment (SwiftGust) [inline CMakeLists.txt]: That would make some problem. ArduPilotPlugin handles almost everything but,
Iris's rotor did not move using ArduPilotPlugin.

Comment (khancyr) [inline CMakeLists.txt]: Yes it does ! You just need to adjust the model see in iris_with_standoffs_demo2 : 
```
<plugin name="arducopter_plugin" filename="libArduPilotPlugin.so">
      <fdm_addr>127.0.0.1</fdm_addr>
      <fdm_port_in>9012</fdm_port_in>
      <fdm_port_out>9013</fdm_port_out>
      <!--
          Require by APM :
          Only change model and gazebo from XYZ to XY-Z coordinates
      -->
      <modelXYZToAirplaneXForwardZDown>0 0 0 3.141593 0 0</modelXYZToAirplaneXForwardZDown>
      <gazeboXYZToNED>0 0 0 3.141593 0 0</gazeboXYZToNED>
      <imuName>iris_demo::iris::iris/imu_link::imu_sensor</imuName>
      <connectionTimeoutMaxCount>5</connectionTimeoutMaxCount>
      <control channel="0">
      <!--
          incoming control command [0, 1]
          so offset it by 0 to get [0, 1]
          and divide max target by 1.
          offset = 0
          multiplier = 838 max rpm / 1 = 838
        -->
        <type>VELOCITY</type>
        <offset>0</offset>
        <p_gain>0.20</p_gain>
        <i_gain>0</i_gain>
        <d_gain>0</d_gain>
        <i_max>0</i_max>
        <i_min>0</i_min>
        <cmd_max>2.5</cmd_max>
        <cmd_min>-2.5</cmd_min>
        <jointName>iris::rotor_0_joint</jointName>
        <multiplier>838</multiplier>
        <controlVelocitySlowdownSim>1</controlVelocitySlowdownSim>
      </control>
      <control channel="1">
        <type>VELOCITY</type>
        <offset>0</offset>
        <p_gain>0.20</p_gain>
        <i_gain>0</i_gain>
        <d_gain>0</d_gain>
        <i_max>0</i_max>
        <i_min>0</i_min>
        <cmd_max>2.5</cmd_max>
        <cmd_min>-2.5</cmd_min>
        <jointName>iris::rotor_1_joint</jointName>
        <multiplier>838</multiplier>
        <controlVelocitySlowdownSim>1</controlVelocitySlowdownSim>
      </control>
      <control channel="2">
        <type>VELOCITY</type>
        <offset>0</offset>
        <p_gain>0.20</p_gain>
        <i_gain>0</i_gain>
        <d_gain>0</d_gain>
        <i_max>0</i_max>
        <i_min>0</i_min>
        <cmd_max>2.5</cmd_max>
        <cmd_min>-2.5</cmd_min>
        <jointName>iris::rotor_2_joint</jointName>
        <multiplier>-838</multiplier>
        <controlVelocitySlowdownSim>1</controlVelocitySlowdownSim>
      </control>
      <control channel="3">
        <type>VELOCITY</type>
        <offset>0</offset>
        <p_gain>0.20</p_gain>
        <i_gain>0</i_gain>
        <d_gain>0</d_gain>
        <i_max>0</i_max>
        <i_min>0</i_min>
        <cmd_max>2.5</cmd_max>
        <cmd_min>-2.5</cmd_min>
        <jointName>iris::rotor_3_joint</jointName>
        <multiplier>-838</multiplier>
        <controlVelocitySlowdownSim>1</controlVelocitySlowdownSim>
      </control>
```

Comment (SwiftGust) [inline CMakeLists.txt]: Uh oh... didn't know that. I'll try that later.

**Pull request closed without merging**